assets/queries/cloudFormation/rds_db_instance_with_deletion_protection_disabled/metadata.json

```diff
@@ -0,0 +1,8 @@
+{
+  "id": "rds_db_instance_with_deletion_protection_disabled",
+  "queryName": "RDS DBInstance With Deletion Protection Disabled",
+  "severity": "LOW",
+  "category": null,
+  "descriptionText": "RDS DBInstance should have deletion protection set to true",
+  "descriptionUrl": "https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-rds-database-instance.html#cfn-rds-dbinstance-deletionprotection"
+}
```

Comment:
[As a good POSIX practice](https://unix.stackexchange.com/a/18789) add end-of-file newline